modules/odc-creating-apiserversource.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * applications/application_life_cycle_management/odc-knative-event-sources.adoc
+
+[id="odc-creating-apiserversource_{context}"]
+= Creating an ApiServerSource event source using the Developer perspective
+
+.Procedure
+
+. Navigate to the **Add** page.
++
+image::event-sources-odc.png[Add an Event Source in the Developer perspective]
+. Select **Event Sources** -> **ApiServerSource**.
++
+image::apiserversource-odc.png[Create an ApiServerSource]
```

Comment:
Probably crop the image to remove the left nav panel?
If you have the screenshot maybe show the other fields you cover in the doc like the Service account name etc. if not this is good enough.
Do we want to ask the ODC devs for input on the commented out sections? If not now, maybe for later?

Reply:
I expanded on the comments, but I don't think these fields are required for now, so I wouldn't block the PR to explain them. It wasn't flagged by the eng SMEs who reviewed it or by QE so I'd say it's OK as is for now.
Maybe add a follow up docs issue in the backlog?

Reply:
I agree, no reason to block this PR. Also, I would go by your judgement Ashleigh since you have tested these scenarios. I will discuss this with the Dev and only if required create an issue.